ccd/deployment.yaml

```diff
@@ -0,0 +1,825 @@
+---
+#
+# NAMESPACE
+#
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ccd
+---
+#
+# IDAM-AUTHENTICATION-WEB CONFIG
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: idam-authentication-web
+  namespace: ccd
+data:
+  IDAM_API_URL: 'http://idam-api'
+  IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_BULK_SCAN_ORCHESTRATOR: '123456'
```

Comment:
if idam stays in AKS then this needs to be an extension point for the helm chart, so that teams add their needed client secret.
the aat one will be fully configured though so no need for this